### ISSUE TITLE
How can I find `pred_dist` function definition?

### ISSUE THREAD

**Issue by H0128Z** [ultralytics/ultralytics#16258]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I'm currenty trying to customize loss function especially DFL loss.

```
class DFLoss(nn.Module):
    """Criterion class for computing DFL losses during training."""

    def __init__(self, reg_max=16) -> None:
        """Initialize the DFL module."""
        super().__init__()
        self.reg_max = reg_max

    def __call__(self, pred_dist, target):
        """
        Return sum of left and right DFL losses.

        Distribution Focal Loss (DFL) proposed in Generalized Focal Loss
        https://ieeexplore.ieee.org/document/9792391
        """
        target = target.clamp_(0, self.reg_max - 1 - 0.01)
        tl = target.long()  # target left
        tr = tl + 1  # target right
        wl = tr - target  # weight left
        wr = 1 - wl  # weight right
        return (
            F.cross_entropy(pred_dist, tl.view(-1), reduction="none").view(tl.shape) * wl
            + F.cross_entropy(pred_dist, tr.view(-1), reduction="none").view(tl.shape) * wr
        ).mean(-1, keepdim=True)
```

from above code, there is `pred_dist` parameter.
I want to know the function definition of `pred_dist`.
Where can I find it?

Thank you in advance for your reply!

**Comment by Y-T-G** [comment]
Should be similar to this, but not rotated.

https://github.com/ultralytics/ultralytics/blob/cc5643e81c390c2f3f314a67b9debf4bbd2ac0da/ultralytics/utils/loss.py#L715
